modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -998,33 +998,34 @@ public void dropTable(String name) {
      * @return Future representing pending completion of the operation.
      */
     private CompletableFuture<List<Table>> tablesAsyncInternal() {
-        List<IgniteUuid> tableIds = directTableIds();
+        return CompletableFuture.supplyAsync(this::directTableIds)
```

Comment:
As I understand, this is a temporary fix to make the method non-blocking, because `directTableIds()` uses `DistributedConfigurationStorage` underneath which performs blocking operations.

Can you please create a ticket and place a `TODO` here?